test/e2e/provisioning/helper.go

```diff
@@ -127,28 +126,27 @@ func runScenarios(t *testing.T, excludeSelector *scenarioSelector, testParams st
 }
 
 func runVerifyTool(manifestPath string, params string) error {
-	homeDir := os.Getenv("HOME")
-	kubeConfigDir := filepath.Join(homeDir, ".kube/config")
+	gopath := os.Getenv("GOPATH")
```

Comment:
nit: usually home dir is the default path for `kubeconfig`.

Reply:
Yeah that is true, but I wanted to allow to run it locally and do not want it to overwrite my kubeconfig

Reply:
how can I run the tests from my local machine ?

Reply:
If you set all env variables, run the `provision_master.sh` once manually you can just execute `go test -tags=e2e -parallel 24 -v -timeout 10m  ./test/e2e/...`

I could probably polish it a little bit more and write a maketarget that does both